src/Task/Ecs/EcsTask.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Wunderio\GrumPHP\Task\Ecs;
+
+use GrumPHP\Runner\TaskResult;
+use GrumPHP\Task\Context\ContextInterface;
+use GrumPHP\Task\Context\GitPreCommitContext;
+use GrumPHP\Task\Context\RunContext;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use GrumPHP\Task\AbstractExternalTask;
+use GrumPHP\Runner\TaskResultInterface;
+use Wunderio\GrumPHP\Task\ContextFileExternalTaskBase;
+
+/**
+ * Class EcsTask.
+ *
+ * @package Wunderio\GrumPHP\Task\Ecs
+ */
+class EcsTask extends ContextFileExternalTaskBase
+{
+  /**
+   * {@inheritdoc}
+   */
+  public function getName(): string
+  {
+    return 'ecs';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getConfigurableOptions(): OptionsResolver
+  {
+    $resolver = new OptionsResolver();
+    $resolver->setDefaults([
+      'ignore_patterns' => ['*/vendor/*','*/node_modules/*', '*/core/*', '*/modules/contrib/*', '*/themes/contrib/*'],
```

Comment:
I wonder if these ignore patterns and other parameter values could be just declared as a class static attribute?

Reply:
Good point. Probably in base class. Repeating patterns, extensions should definitely be added in single place.